File: src/awt2swing/Label.java
package awt2swing;

import javax.swing.JLabel;

public class Label extends JLabel {

	public Label() {
		super();
	}
	
	public Label(String text) {
		super(text);
	}
	
	public Label(String text, int center) {
		super(text, center);
	}

	public void setAlignment(int alignment) {
		setAlignmentX(alignment);
		
	}

}

File: site/swingjs/j2s/swingjs/awt/Label.js
Clazz.declarePackage ("swingjs.awt");
Clazz.load (["javax.swing.JLabel"], "swingjs.awt.Label", null, function () {
c$ = Clazz.declareType (swingjs.awt, "Label", javax.swing.JLabel);
Clazz.defineMethod (c$, "setAlignment", 
function (alignment) {
this.setAlignmentX (alignment);
}, "~N");
});
